src/app/events/MessageCreate.ts

```diff
@@ -5,24 +5,33 @@ import { DiscordEvent } from '../types/discord/DiscordEvent';
 import MessageCreateOnDEGEN from './chat/MessageCreateOnDEGEN';
 import ServiceUtils from '../utils/ServiceUtils';
 import { LogUtils } from '../utils/Log';
+import HandleAFK from './chat/HandleAFK';
 
 export default class implements DiscordEvent {
 	name = 'messageCreate';
 	once = false;
 
 	async execute(message: Message): Promise<any> {
 		try {
-			if(message.author.bot) return;
-
+			if (message.author.bot) return;
+			// Check mentions for AFK users
+			if (message.mentions.users.size > 0) {
+				await HandleAFK(message).catch((e) => {
+					LogUtils.logError('DEGEN failed to handle AFK', e);
+				});
+			}
```

Comment:
Can you gate this to only run if the message is on the BanklessDAO or garage server. I'm assuming this is only a Bankless feature as the commands are restricted to Bankless servers.

Reply:
@jonvaljonathan please move this code to be inside the Bankless block of code